app/templates/containerdetails.html

```diff
@@ -361,7 +361,7 @@ <h3 class="pull-right">
             <strong uib-tooltip="Date and time of the last update to the Tool entry">Last Update</strong>:
             {{ containerObj.lastUpdated ? getDateTimeString(containerObj.lastUpdated) : 'n/a' }}
           </li>
-          <li>
+          <li ng-show="containerObj.description">
```

Comment:
Let's be a bit more helpful, let's link to the description of a doc field in the cwl documentation (when CWL) and the equivalent for WDL